flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/converters/SqlNodeConvertUtils.java

```diff
@@ -63,6 +69,32 @@ static CatalogView toCatalogView(
         // This bug is fixed in CALCITE-3877 of Calcite 1.23.0.
         String originalQuery = context.toQuotedSqlString(query);
         SqlNode validateQuery = context.getSqlValidator().validate(query);
+
+        // Check name is unique.
```

Comment:
I would suggest refactoring the code into a private method for better readability, i.e. clean code.